arcus_manifests/common/queries.py

```diff
@@ -5,8 +5,21 @@
 
 def file_query(file_list):
     query = f"""
```

Comment:
Please add a docstring describing this function, it's purpose, and design decisions. e.g. why protocol is being set to what it is, etc. 

e.g:

```suggestion
def file_query(file_list):
   """Build query for File Manifest
   
   Builds the query text for the file manifest. Sets the protocol to be [xyz]. See 
   https://github.com/d3b-center/d3b-arcus-manifest-prep/blob/main/docs/design/non-technical-transformation.md#file-manifest 
   for description of the file manifest output. 
   
    :param file_list: List of files to include in the file manifest
    :type file_list: str
    :return: Text query to generate the file manifest
    :rtype: str
   """
    query = f"""
```